distribution/src/bin/elasticsearch-cli.bat

```diff
@@ -0,0 +1,22 @@
+call "%~dp0elasticsearch-env.bat" || exit /b 1
+
+if defined ES_ADDITIONAL_SOURCES (
+  for %%a in ("%ES_ADDITIONAL_SOURCES:;=","%") do (
+    call %~dp0%%a
+  )
+)
+
+for /f "tokens=1*" %%a in ("%*") do (
+  set main_class=%%a
+  set arguments=%%b
+)
+
+%JAVA% ^
+  %ES_JAVA_OPTS% ^
+  -Des.path.home="%ES_HOME%" ^
+  -Des.path.conf="%ES_PATH_CONF%" ^
+  -Des.distribution.flavor="%ES_DISTRIBUTION_FLAVOR%" ^
+  -Des.distribution.type="%ES_DISTRIBUTION_TYPE%" ^
+  -cp "%ES_CLASSPATH%" ^
+  %main_class% ^
```

Comment:
seems a little weird to be switching between all cap vars and all lowercase vars

Reply:
I am trying to distinguish between variables set outside the script and within the script that will not leave the script.